src/components/PostCard/PostDetails/PostDetails.scss

```diff
@@ -73,6 +73,19 @@
   }
 }
 
+.timestamp {
+  // KMT: composes seems to be ignored, need help to understand what I'm doing wrong
```

Comment:
Ignored or over-written? I don't have a lot of experience using the `composes` keyword either